packages/side-nav/src/Search/Search.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { css } from "emotion";
 import ThemeContext from "@hig/theme-context";
-import { Search24, ClearSmall24 } from "@hig/icons";
+import { Search24, Error24 } from "@hig/icons";
```

Comment:
ClearSmall24 has been removed in Icons 3.0. For now I just use Error24, as it looks the most the same with the previous ClearSmall24. Let me know if we should use other icons for this.

Reply:
And for the Search component in SideNav, all the height/size are hard-coded, the entire Search component currently does not have responsive UI styles for density changes. Thus, the icons here are only 24 sizes. (change icon to 16 but the search box still use hard-code 30px height in high-density will not look good) 😞

Reply:
I think this is due for an update coming up, so we can leave that for now.